core/domain/summary_services_test.py

```diff
@@ -504,3 +506,38 @@ def test_for_top_rated_explorations(self):
                            top_rated_exploration_summaries]
 
         self.assertEqual(expected_ordering, actual_ordering)
+
+
+    def test_all_top_rated_explorations_have_ratings(self):
```

Comment:
test_only_explorations_with_ratings_are_returned

(also, remove one newline from above this. It's not a top-level fn)